### PR TITLE
Updated example in xviz-builder.md

### PR DESCRIPTION
**Pull request by MGraefe**
Example now works out of the box with npm packages.

### CODE REVIEW

**Comment by xintongxia** [comment]
Thanks for your contribution.

File: docs/api-reference/xviz-builder.md [hunk 1]
```diff
@@ -6,39 +6,35 @@ provides convenient chaining methods to build up XVIZ data in an easy-to-read wa
...
-    yaw: 0.123
-  })
+  .pose()
+  .timestamp(123)
+  .mapOrigin(12.345, 12.345, 12.345)
+  .position(123, 123, 123)
+  .orientation(0.123, 0.123, 0.123)
 
+xvizBuilder
   .variable('/velocity')
```
Comment (xintongxia) [inline docs/api-reference/xviz-builder.md]: Our documentation here is not quite accurate.

'/vehicle/velocity' is supposed to be a `timeSeries` stream.
 
```
xvizBuilder
  .timeSeries('/vehicle/velocity')
```

Here is the schema detail about `TimeSeries` and `Variables`.

[TimeSeries](https://github.com/uber/xviz/blob/master/docs/protocol-schema/core-types.md#time-series-state)
[Variable](https://github.com/uber/xviz/blob/master/docs/protocol-schema/core-types.md#variables)

File: docs/api-reference/xviz-builder.md [hunk 2]
```diff
@@ -6,39 +6,35 @@ provides convenient chaining methods to build up XVIZ data in an easy-to-read wa
 ## Example
 
 ```js
-import {XVIZBuilder} from '@xviz/builder';
+import { XVIZBuilder } from '@xviz/builder';
```
Comment (xintongxia) [inline docs/api-reference/xviz-builder.md]: remove the extra space?

File: docs/api-reference/xviz-builder.md (hunk 1 above)
Comment (MGraefe) [reply docs/api-reference/xviz-builder.md]: Makes sense, I was confused why variable would require an array anyway. Will fix this once I get back to work tomorrow.

File: docs/api-reference/xviz-builder.md (hunk 2 above)
Comment (MGraefe) [reply docs/api-reference/xviz-builder.md]: Will do

**Comment by MGraefe** [comment]
Fixed some mentioned things in a new commit.
@twojtasz I actually tested the exact example code with node.js

**Pull request merged**